source/uri-options/uri-options.rst

```diff
@@ -71,6 +71,14 @@ is specified with an SRV URI, because the URI may resolve to multiple
 hosts. The driver MUST allow specifying ``directConnection=false`` URI
 option with an SRV URI.
 
+Non-SRV URI with srvServiceName URI option
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The driver MUST report an error if the ``srvServiceName`` URI option is
+specified with a non-SRV URI (i.e. ``mongodb://``) because SRV lookup
+only occurs with SRV URIs. The driver MUST allow specifying the ``srvServiceName``
+URI option with an SRV URI.
```

Comment:
Personally, I wouldn't worry about this.  I think it's OK for the option to be ignored if not needed.